### DISCUSSION TITLE
Display unit price of $Alph and other asset on mobile wallet

### DISCUSSION THREAD

**Discussion by basedev007** [alephium/alephium-frontend#808] [Feature requests]
Currently you only see the total worth of $Alph on the mobile wallet, it will be very helpful if the unit price can also be displayed including other assets being held in the wallet 
For instance 
$Alph 

Price : $1.8
Holding: 500( $900)

**Comment by nop33** [comment]
That's already the case, the total worth is showing on the top, and the worth of each token is showing in the list.

**Comment by nop33** [comment]
In the current latest version of the app (2.1.2) the price of a token is shown once you tap on it.

Closing discussion.
